spec/jobs/reports/sp_user_counts_report_spec.rb

```diff
@@ -10,6 +10,46 @@
     expect(subject.perform(Time.zone.today)).to eq('[]')
   end
 
+  it 'logs to analytics' do
+    ServiceProvider.create(issuer: issuer, friendly_name: issuer, app_id: app_id)
+    ServiceProviderIdentity.create(user_id: 1, service_provider: issuer, uuid: 'foo1')
+    freeze_time do
+      timestamp = Time.zone.now.iso8601
+      log_hash = {
+        app_id: app_id,
+        ial1_user_total: 1,
+        ial2_user_total: 0,
+        issuer: issuer,
+        name: Analytics::REPORT_SP_USER_COUNTS,
+        time: timestamp,
+        user_total: 1,
+      }
+      allow(subject).to receive(:write_hash_to_reports_log).with(log_hash)
+      log_hash1 = {
+        name: Analytics::REPORT_REGISTERED_USERS_COUNT,
+        time: timestamp,
+        count: 0,
+      }
+      allow(subject).to receive(:write_hash_to_reports_log).with(log_hash1)
```

Comment:
let's inline these variables?
```suggestion
      allow(subject).to receive(:write_hash_to_reports_log).with(
        app_id: app_id,
        ial1_user_total: 1,
        ial2_user_total: 0,
        issuer: issuer,
        name: Analytics::REPORT_SP_USER_COUNTS,
        time: timestamp,
        user_total: 1,
      )
      allow(subject).to receive(:write_hash_to_reports_log).with(
        name: Analytics::REPORT_REGISTERED_USERS_COUNT,
        time: timestamp,
        count: 0,
      )
```